nengo_gui/static/sim_control.css

```diff
@@ -9,7 +9,8 @@
     flex: 0 0 auto;
     -webkit-flex: 0 0 auto;
     width: 100%;
-    z-index:99999999;
+    z-index:99999998; /* The SVG elements in the netgraph use z-index, so make
+                         sure they never cover the control panel */
```

Comment:
It is easy to miss that this number ends in 8. Maybe point at that in the comment (and say why)?

Reply:
I updated the comment accordingly. Would be nice to get rid of such magic numbers.

Reply:
Yeah, but how?

Reply:
🤔

Reply:
Just checked the DOM again. I don't see any need for the z-index at all (the comment is based on what Terry told me). The netgraph is contained within a DOM element with `overflow: hidden` and there are no elements with `position: fixed`, so these are two completely separate object stacks.